tests/test_image.py

```diff
@@ -281,3 +283,14 @@ def testMultiDimBroadcast():
     mdim_ims_np, mdim_ims = get_mdim_images()
     X = mdim_ims + ims
     assert np.allclose(X[0], 2 * ims.asnumpy())
+
+
+def testShow():
+    """
+    Test show doesn't crash.
+    """
+    # Use non GUI backend.
+    matplotlib.use("Agg")
```

Comment:
Stuff into a decorator? May need to do this more if we have new display functionality.

Reply:
(cracks knuckles) my pleasure

Reply:
Nice!